src/clarity-angular/layout/nav/_header.clarity.scss

```diff
@@ -375,26 +362,12 @@ $clr-header-nav-text-horizontal-padding: $clr_baselineRem_1;
                 //for the height of the header.
                 .dropdown-menu {
                     margin-top: -1 * $clr_baselineRem_0_375;
-
-                    .dropdown-menu {
```

Comment:
@jeeyun - just double checking that this is the correct itme to remove this. Can you confirm that we no longer need it?